common/dna_data/wc_easter_egg_dna.txt

```diff
@@ -0,0 +1,1109 @@
+﻿aliza_dna = { # by ercarp
```

Comment:
Give some kind of hair to Ion.

Reply:
Added in 89df74832164f9b4498d3f14e2c00f3ec4913b9b.